src/io.cxx

```diff
@@ -58,4 +59,65 @@ std::ostream &operator<<(std::ostream &out, DDouble x)
     return out;
 }
 
+XPREC_API_EXPORT
+std::string to_string(DDouble d, size_t nDigits){
+	using namespace std;
+
+	//Get some basic cases out of the way.
+	if(d.hi() == 0.0)
+		return "0.0";
+
+	if(isnan(d.hi()))
+		return "NaN";
+
+	if(isinf(d.hi()))
+		if(d.hi() > 0.0)
+			return "Inf";
+		else
+			return "-Inf";
+
+
+	//Alright. Get a string started. Handle sign.
+	string s;
+
+	if(d.hi() < 0.0) s += "-";
```

Comment:
Please do not do this.

There is a clang format file in the repo. You can use `clang-format -i src/io.cxx` to automatically format the file according to the spect.

Reply:
Don't do what?

Reply:
one-line if's, please split them across lines.